src/ResourceManager/OperationalInsights/Commands.OperationalInsights/DataSources/GetAzureOperationalInsightsDataSourceCommand.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.Azure.Commands.OperationalInsights
 {
-    [Cmdlet(VerbsCommon.Get, Constants.DataSource, DefaultParameterSetName = ByWorkspaceName), OutputType(typeof(List<PSDataSource>), typeof(PSDataSource))]
+    [Cmdlet(VerbsCommon.Get, Constants.DataSource, DefaultParameterSetName = ByWorkspaceNameByKind), OutputType(typeof(List<PSDataSource>), typeof(PSDataSource))]
```

Comment:
this is actually a fix I spotted, we have 4 parameterSet for this cmdlet, ByWorkspaceName isn't a valid parameterSet for this cmdlet. (probably copied from other cmdlets )

ByWorkspaceObjectByName {-Workspace -Name}
ByWorkspaceObjectByKind {-Workspace -Kind}
ByWorkspaceNameByName {-ResourceGroup -WorkspaceName -Name}
ByWorkspaceNameByKind {-ResourceGroup -WorkspaceName -Kind}